multiplex-rs/src/lib.rs

```diff
@@ -335,21 +339,35 @@ impl<T: AsyncRead + AsyncWrite> StreamMuxer for Multiplex<T> {
     }
 }
 
+pub type MultiplexConfig = BufferedMultiplexConfig<[u8; 0]>;
```

Comment:
I find it annoying that you can no longer build a config by just writing `let cfg = MultiplexConfig;`. Instead you have to do `MultiplexConfig::default()`. But that's not a big problem.